sig-scheduling/annual-report-2021.md

```diff
@@ -4,15 +4,15 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Efficient re-queueing of pods, significantly cutting the number of failed scheduling cycles
+   - Improvements to preemption performance
+   - Simplified plugin configuration in component config
 
```

Comment:
- Usability improvement of Scheduler Framework (unambiguous lock semantics, derivable SharedInformerFactory and kubeConfig)

Reply:
I don't know if those are worth mentioning, seems like normal cleanups.

Reply:
Fine with me not mentioning the details.